docs/projects/modules/gsoc/pages/2021/project-ideas/plugin-installation-manager-tool.adoc

```diff
@@ -13,7 +13,7 @@ skills:
 - Package management tool theory
 mentors:
 - "markewaite"
-- "freyam"
+- "jonesbusy"
```

Comment:
As this is the project idea page for 2021 if I remember correctly it should be neither "freyam" nor "jonesbusy". Can you please check again?